package.json

```diff
@@ -5,7 +5,7 @@
   "private": "true",
   "dependencies": {
     "autoprefixer": "10.4.7",
-    "postcss": "8.4.13",
```

Comment:
This should handle case sensitivity issues to be compatible with Windows. See https://googlecloudplatform.github.io/samples-style-guide/#portability.